pygmt/src/blockm.py

```diff
@@ -14,18 +14,18 @@
 
 def _blockm(block_method, table, outfile, x, y, z, **kwargs):
     r"""
-    Block average (x,y,z) data tables by mean or median estimation.
+    Block average (x,y,z) data tables by mean, median, or mode estimation.
 
     Reads arbitrarily located (x,y,z) triples [or optionally weighted
-    quadruples (x,y,z,w)] from a table and writes to the output a mean or
-    median (depending on ``block_method``) position and value for every
+    quadruples (x,y,z,w)] from a table and writes to the output a mean,
+    median, or mode (depending on ``block_method``) position and value for every
     non-empty block in a grid region defined by the ``region`` and ``spacing``
     parameters.
```

Comment:
```suggestion
    median, or mode (depending on ``block_method``) position and value for
    every non-empty block in a grid region defined by the ``region`` and
    ``spacing`` parameters.
```
Just a small formatting fix for line-length.